File: app/src/main/java/za/co/duartbreedt/androidconceptsreference/EmployeeData.java
package za.co.duartbreedt.androidconceptsreference;

import com.google.gson.annotations.SerializedName;

import java.io.Serializable;

public class EmployeeData implements Serializable {

    private int id;

    private String email;

    @SerializedName("first_name")
    private String firstName;

    @SerializedName("last_name")
    private String lastName;

    private String avatar;

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public String getEmail() {
        return email;
    }

    public void setEmail(String email) {
        this.email = email;
    }

    public String getFirstName() {
        return firstName;
    }

    public void setFirstName(String firstName) {
        this.firstName = firstName;
    }

    public String getLastName() {
        return lastName;
    }

    public void setLastName(String lastName) {
        this.lastName = lastName;
    }

    public String getAvatar() {
        return avatar;
    }

    public void setAvatar(String avatar) {
        this.avatar = avatar;
    }
}

File: app/src/main/java/za/co/duartbreedt/androidconceptsreference/mvvm/model/domain/EmployeeData.kt
package za.co.duartbreedt.androidconceptsreference.services.employee.domain

import com.google.gson.annotations.SerializedName

data class EmployeeData(
    val id: Int,
    val email: String,
    @SerializedName("first_name")
    val firstName: String,
    @SerializedName("last_name")
    val lastName: String,
    val avatar: String,
)